README.md

```diff
@@ -1,6 +1,6 @@
 # Kubernetes Cluster API Provider AWS
 
-[![Go Report Card](https://goreportcard.com/badge/sigs.k8s.io/cluster-api-provider-aws)](https://goreportcard.com/report/sigs.k8s.io/cluster-api-provider-aws)
+[![Go Report Card](https://goreportcard.com/badge/kubernetes-sigs/cluster-api-provider-aws)](https://goreportcard.com/report/kubernetes-sigs/cluster-api-provider-aws)
```

Comment:
Can you please explain this change? Not sure what the difference is.

Reply:
Go report card doesn't resolve sigs.k8s.io. If we want to navigate to a line where an issue is reported, this change is required.